torch/_export/__init__.py

```diff
@@ -118,9 +118,6 @@ def capture_pre_autograd_graph(
 
     capture_pre_autograd_graph_warning()
 
-    if sys.platform == "win32":
-        raise RuntimeError("capture_pre_autograd_graph not yet supported on Windows")
-
```

Comment:
We don't intend to expand support surface the deprecated API like capture_pre_autograd(), and it will be error prone to use this in the future. 

Can you elaborate what's the issue with torch.export.export?

Reply:
Capture_pre_autograd won't work on windows platforms because it uses dynamo internally. So I feel like even if you remove this check, you will still hit an error and if you don't, it just means it accidentally worked. So it is unreliable i think.